ipatests/test_integration/test_epn.py

```diff
@@ -209,6 +209,35 @@ def uninstall(cls, mh):
         cls.master.run_command(r'rm -f /etc/pki/tls/private/postfix.key')
         cls.master.run_command(r'rm -f /etc/pki/tls/certs/postfix.pem')
 
+    def test_EPN_config_file(self):
```

Comment:
This test needs a wrapping around platforms it can work on. It will fail on Debian/Ubuntu as it uses direct RPM commands. 
```
@pytest.mark.skip_if_platform(
    "debian", reason="Cannot check installed packages using RPM"
)

```

would allow to proceed here.

Reply:
Good catch, fixed, thank you.